### PR TITLE
Pipeline summary

### PR DESCRIPTION
**Pull request by marissaDubbelaar**
Increment the readme of the page with a `pipeline summary` and increment the `documentation`

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/mhcquant _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,10 +59,65 @@ On release, automated continuous integration tests run the pipeline on a full-si
                                  --outdir ./results
    ```
 
+## Pipeline summary
+
+### Default Steps
+
+By default the pipeline currently performs the following
```
Comment (SusiJo) [inline README.md:66]: All steps are listed below additional steps. Shouldn't there be a few default steps?

**Review by SusiJo — COMMENTED** [review]
LGTM in general!

File: README.md (hunk 1 above)
Comment (marissaDubbelaar) [reply README.md:66]: Apparently I removed this with the enthusiasm that I had.
I included this again!

Comment (SusiJo) [reply README.md:66]: Perfect, looks great now!

**Review by SusiJo — APPROVED** [review]

**Pull request merged**